javascript/javascript1/week4/tictactoeApp.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+         const position = [
+ ['o', ' ', ' '],
+ ['o', ' ', ' '],
+ ['o', ' ', ' ']
+ ];
+
+const position = [
```

Comment:
You can't reuse the same variable name for a const in the same scope. I would choose a different name for every `position` variable!